client/src/pages/private/ParticipantTable.js

```diff
@@ -600,6 +600,9 @@ export default () => {
             onSubmit={async (values) => {
               if (values.phoneNumber && Number.isInteger(values.phoneNumber))
                 values.phoneNumber = values.phoneNumber.toString();
+              if (values.postalCode && values.postalCode.length > 3) {
```

Comment:
should this be >= in the case of them inputting only the three characters of the postal code?